GeanyVC: build is failing at linking the unittests due to missing gtkspell lib

Looks like the recent build of [90974ab] is failing at linking geanyvc unittests due to missing `gtkspell` lib:

[jobs/685847261](https://travis-ci.org/github/geany/geany-plugins/jobs/685847261#L7195)

```
...
  CCLD     unittests
unittests-geanyvc.o: In function `vccommit_activated':
/home/travis/build/geany/geany-plugins/_build/geany-plugins-1.37/_build/geanyvc/src/../../../geanyvc/src/geanyvc.c:1739: undefined reference to `gtkspell_new_attach'
collect2: error: ld returned 1 exit status
make[4]: *** [unittests] Error 1
make[4]: Leaving directory `/home/travis/build/geany/geany-plugins/_build/geany-plugins-1.37/_build/geanyvc/src'
make[3]: *** [check-am] Error 2
make[3]: Leaving directory `/home/travis/build/geany/geany-plugins/_build/geany-plugins-1.37/_build/geanyvc/src'
....
```

Fix is submitted in  PR #977.

--- a/geanyvc/src/Makefile.am
+++ b/geanyvc/src/Makefile.am
@@ -30,7 +30,7 @@ TESTS = unittests
 check_PROGRAMS = unittests
 unittests_SOURCES = unittests.c $(geanyvc_la_SOURCES)
 unittests_CFLAGS  = $(GEANY_CFLAGS) $(geanyvc_la_CFLAGS) -DUNITTESTS
-unittests_LDADD   = @GEANY_LIBS@ $(INTLLIBS) @CHECK_LIBS@
+unittests_LDADD   = @GEANY_LIBS@ $(INTLLIBS) $(geanyvc_la_LIBADD) @CHECK_LIBS@
 endif
 
 include $(top_srcdir)/build/cppcheck.mk

